+dot support
+symbols index
+fuzzy search
+build support
+git support
+location list
--- 2022-10-24
+bitmap font
--- 2022-10-22
-tilda support
+full insert undo
	enable undo if text is different
	remove modified bool
--- 2022-10-21
-e-ink color scheme
-space whitespace mark
